addons/point_of_sale/models/pos_payment_method.py

```diff
@@ -3,56 +3,52 @@
-        default=lambda self: self.env.company.account_default_pos_receivable_account_id,
         ondelete='restrict',
-        help='Account used as counterpart of the income account in the accounting entry representing the pos sales.')
-    is_cash_count = fields.Boolean(string='Cash')
-    cash_journal_id = fields.Many2one('account.journal',
-        string='Cash Journal',
-        domain=[('type', '=', 'cash')],
+        domain=[('reconcile', '=', True), ('user_type_id.type', '=', 'receivable')],
+        help="Leave empty to use the default account from the company setting.\n"
+             "Overrides the company's receivable account (for Point of Sale) used in the journal entries.")
+    is_cash_count = fields.Boolean(string='Cash', compute="_compute_is_cash_count", store=True)
+    journal_id = fields.Many2one('account.journal',
+        string='Journal',
+        domain=[('type', 'in', ('cash', 'bank'))],
         ondelete='restrict',
-        help='The payment method is of type cash. A cash statement will be automatically generated.')
+        help='Leave empty to use the receivable account of customer.\n'
+             'Defines the journal where to book the accumulated payments (or individual payment if Identify Customer is true) after closing the session.\n'
+             'For cash journal, we directly write to the default account in the journal via statement lines.\n'
+             'For bank journal, we write to the outstanding account specified in this payment method.')
     split_transactions = fields.Boolean(
-        string='Split Transactions',
+        string='Identify Customer',
         default=False,
-        help='If ticked, each payment will generate a separated journal item. Ticking that option will slow the closing of the PoS.')
+        help='Forces to set a customer when using this payment method and splits the journal entries for each customer. It could slow down the closing process.')
     open_session_ids = fields.Many2many('pos.session', string='Pos Sessions', compute='_compute_open_session_ids', help='Open PoS sessions that are using this payment method.')
     config_ids = fields.Many2many('pos.config', string='Point of Sale Configurations')
     company_id = fields.Many2one('res.company', string='Company', default=lambda self: self.env.company)
     use_payment_terminal = fields.Selection(selection=lambda self: self._get_payment_terminal_selection(), string='Use a Payment Terminal', help='Record payments with a terminal on this journal.')
     hide_use_payment_terminal = fields.Boolean(compute='_compute_hide_use_payment_terminal', help='Technical field which is used to '
                                                'hide use_payment_terminal when no payment interfaces are installed.')
     active = fields.Boolean(default=True)
+    type = fields.Selection(selection=[('cash', 'Cash'), ('bank', 'Bank'), ('pay_later', 'Customer Account')], compute="_compute_type")
```

Comment:
maybe we can store it and remove the 'is_cash_count'? (so load it in models.js of point of sale instead of the one in pos_settle_due)